specs/phase0/p2p-interface.md

```diff
@@ -397,6 +398,8 @@ The following validations MUST pass before forwarding the `attestation` on the s
   that is, it has exactly one participating validator (`len([bit in bit attestation.aggregation_bits if bit]) == 1`, i.e. exactly 1 bit is set).
 - _[IGNORE]_ There has been no other valid attestation seen on an attestation subnet
   that has an identical `attestation.data.target.epoch` and participating validator index.
+- _[REJECT]_ The number of aggregation bits matches the committee size -- i.e.
```

Comment:
I would put this before the "The attestation is unaggregated" condition. It's essentially a well-formedness condition on the datatype and then that the datatype contains the expected value immediately after

Reply:
actually, this is arguable a more expensive operation that checking for the unaggregated condition. Maybe put it right after. But I would definitely put it before the "There has been no other valid attestation..."